src/components/NewPosition/RangeSelector/RangeSelector.tsx

```diff
@@ -74,7 +82,7 @@ export const RangeSelector: React.FC<IRangeSelector> = ({
 
```

Comment:
```Typescript
  const [leftRange, setLeftRange] = useState(MIN_TICK)
  const [rightRange, setRightRange] = useState(MAX_TICK)
  ```

Overall, `MAX_TICK` and `MIN_TICK` shouldn't be used at all. Instead, use `getMaxTick(tickSpacing)` and `getMinTick(tickSpacing)`

Reply:
applied